assets/js/base/context/hooks/use-checkout-submit.js

```diff
@@ -54,16 +52,10 @@ export const useCheckoutSubmit = () => {
 	const waitingForProcessing =
 		isProcessing || isAfterProcessing || isBeforeProcessing;
 	const waitingForRedirect = isComplete && ! hasError;
-	const defaultLabel =
-		paymentMethod.placeOrderButtonLabel ||
-		__( 'Place Order', 'woo-gutenberg-products-block' );
-	const label = __experimentalApplyCheckoutFilter( {
-		filterName: 'placeOrderButtonLabel',
-		defaultValue: defaultLabel,
-	} );
+	const submitButtonText = paymentMethod.placeOrderButtonLabel;
```

Comment:
Can we rename this? (See the comment further down)

Reply:
I've renamed this when I renamed the other section, that you mentioned.